src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -292,5 +296,26 @@
   
   <!-- This target collects all the resolved references that are used to actually compile. -->
   <Target Name="CollectResolvedCompilationReferencesDesignTime" DependsOnTargets="CompileDesignTime" Returns="@(ReferencePathWithRefAssemblies)" />
+  
+  <!-- This target collects all the things built by the project for the up to date check. -->
+  <!-- See CopyFileToOutputDirectory target -->
+  <Target Name="CollectBuiltDesignTime" DependsOnTargets="CompileDesignTime" Returns="@(UpToDateCheckBuilt)">
```

Comment:
@nguerrera @rainersigwald @dsplaisted @andygerlicher - Can you guys take a look at this and see what you think about the choice of items/properties to use?

Reply:
Looks reasonable to me. It's unfortunate that we'll now have to keep these conditions in sync with the common targets ones, but I can't think of a better way.